renderer/components/Home/WalletSettingsFormLocal.js

```diff
@@ -27,6 +34,12 @@ class WalletSettingsFormLocal extends React.Component {
     })
   }
 
+  resetNeutrinoSettings = () => {
+    const { formApi, wallet } = this.props
+    const { chain, network } = wallet
+    formApi.setValue('neutrinoNodes', config.lnd.neutrino[chain][network])
```

Comment:
The behaviour here is a little different to other elements of the wallet config and potentially problematic. The way that the autopilot settings work for example, is as default overrides - meaning that if you have customised the settings then they are saved in the database but if you have not or you have set them back to the defaults then they are removed from the database.

That ensures that if we update our defaults in a future release, users will automatically assume those new defaults unless they have specifically overridden the settings and set them to some non-default values.

However with the neutrino configs they are saved into the database and never removed again even after restoring the defaults. This could be problematic as it could mean that users that have simply changed the name of their wallet (for example) would no longer receive changes to our default configs if we update them for a future release (since they would still have hard coded overrides in their database) and they could be left stuck with a connection to a retired node.

Reply:
Made a number of improvements towards this goal